src/apollo/gql/gqlHomePage.ts

```diff
@@ -66,7 +90,6 @@ export const FETCH_HOMEPAGE_DATA = gql`
 					image
 				}
 			}
-			count
```

Comment:
By removing this field, does it not create inconsistency between schemas?

Reply:
In GraphQl you can choose which fields to fetch, and you can choose not to fetch the count

Reply:
Oh, I thought this is the server-side, yes that is correct